api/handlers.go

```diff
@@ -553,6 +691,17 @@ func (si *ServerImplementation) LookupTransaction(ctx echo.Context, txid string)
 // SearchForTransactions returns transactions matching the provided parameters
 // (GET /v2/transactions)
 func (si *ServerImplementation) SearchForTransactions(ctx echo.Context, params generated.SearchForTransactionsParams) error {
+	rc, dpName := Verify(si.disabledParams, "SearchForTransactions", ctx, si.log)
+	switch rc {
+	case verifyIsGood:
+		// Do nothing
+		break
+	case verifyFailedParameter:
+		return badRequest(ctx, fmt.Sprintf("provided disabled parameter: %s", dpName))
+
+	case verifyFailedEndpoint:
+		return badRequest(ctx, "endpoint is disabled")
+	}
```

Comment:
would it be better to put this check in a helper method to reduce repeated code?

Reply:
I tried going through some iterations of that but I couldn't really get to a solution I liked very much.  Do you have some suggestions?  The problem is that you always want to end up differentiating between a failed parameter and a failed endpoint so you always end up with some sort of switch statement.  

You could do something where it returns true or false, but then you have to handle the error generated by badRequest.

Reply:
maybe something like, 
`if rc>0 {
return getErrorMsg(rc)
}
`
where getErrorMsg(rc) returns the different badRequest messages.

Reply:
You can do this middleware/plugin pattern used by go's HTTP (and echo). The generated echo code abstracts this a bit so I don't think it would be as clean. The middleware version would be similar but requires figuring out the operationID based on the context. Problematic because swagger has things like `/v2/assets/{assetid}` and echo uses `/v2/assets/:assetid:`.

The basic pattern looks something like this, where you chain the handlers together:
```
func verifyHandler(operationID, log logger.Log, next echo.Handler) echo.Handler {
    return func(e echo.Context) error {
        if rc := verify(operationID, e, log); rc != 0 {
            switch { ...returns here... }
        }
        return next(e)
    }
}
```

Something similar where you call badRequest in a helper then return the error code or run the rest of the handler would be functionally identical but less idiomatic

Reply:
I've implemented this although I had to use reflection because each of the functions had a different interface.  Make sure to "hide whitespace" when reviewing in github.